client/components/seo/preview-upgrade-nudge/index.jsx

```diff
@@ -112,15 +157,15 @@ const SeoPreviewNudge = ( { translate, domain, plan = {}, businessPlan = {} } )
 				</PlanCompareCard>
 				<PlanCompareCard
 					title={ businessPlan.product_name_short || '' }
-					line={ translate( '%(price)s per month, billed yearly', { args: { price: businessPlanPrice } } ) }
+					line={ priceInfo }
 					buttonName={ translate( 'Upgrade' ) }
 					onClick={ () => {
 						recordTracksEvent( 'calypso_upgrade_nudge_cta_click', {
 							cta_size: 'expanded',
 							cta_name: 'calypso_seo_preview_upgrade_nudge',
 							cta_feature: FEATURE_ADVANCED_SEO
 						} );
-						page( '/checkout/' + domain + '/business' );
+						page( '/checkout/' + domain + ( isJetpack ? '/professional' : '/business' ) );
```

Comment:
I know this is not coming from this PR, but this will fail for sites that are installed in subdirectories. Let's make sure to use the site slug instead, or at least replace forward slashes `/` with double colons `::` in the `domain`.